### ISSUE TITLE
kirbyup fails silently

### ISSUE THREAD

**Issue by felixfischer** [johannschopplich/kirbyup#12]
Errors that are not PrettyErrors are silently discarded:

https://github.com/johannschopplich/kirbyup/blob/6807908ea9ccfbec75e922619c8d249e382a8f1e/src/node/errors.ts#L17

I ran into problems because I had no postcss.config.js in my plugin folder. This caused kirbyup to look for the file higher up in the tree, where it found an incompatible version of it.

I had to add `else { console.log(error) }` to find out what went wrong.

**Comment by johannschopplich** [comment]
I see. Could you please test the following change and let me know if your error does pile up?

```js
if (error instanceof Error) {
  consola.error(error.message)
}
```

**Comment by johannschopplich** [comment]
Fixed in [v0.23.0](https://github.com/johannschopplich/kirbyup/commit/6e88beacf17e65a44ebaa98c064555ee5892e0e7).

Now, all errors will be thrown as of v0.23.0+. Thanks for reporting!

**Comment by felixfischer** [comment]
> Could you please test the following change and let me know if your error does pile up?

That worked! Thanks.
